samples/Constraint/availability_constraint.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<mes:Structure xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xml="http://www.w3.org/XML/1998/namespace" xmlns:mes="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message" xmlns:str="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/structure" xmlns:com="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/common" xsi:schemaLocation="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message ../../schemas/SDMXMessage.xsd">
+	<mes:Header>
+		<mes:ID>IREF583870</mes:ID>
+		<mes:Test>false</mes:Test>
+		<mes:Prepared>2024-07-30T16:53:05Z</mes:Prepared>
+		<mes:Sender id="Unknown"/>
+		<mes:Receiver id="not_supplied"/>
+	</mes:Header>
+	<mes:Structures>
+		<str:AvailabilityConstraints>
+			<str:AvailabilityConstraint>
+				<com:Annotations>
+					<com:Annotation id="obs_count">

```

Comment:
Obs_count and series_count should now be attributes.

Reply:
done